src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -238,6 +238,21 @@ private IQueryable<Instance> BuildQueryFromParameters(Dictionary<string, StringV
                             queryBuilder = QueryBuilderExcludeConfirmedBy(queryBuilder, queryValue);
                             break;
                         case "language":
+                            break;
+                        case "status.isHardDeleted":
+                            bool isHardDeleted = bool.Parse(queryValue);
```

Comment:
What happens here if the value is different from `true` and `false`?

Reply:
There is no additional handling of isComplete bool further up, so I figured we wanted to let an exception be thrown and then it is handled elsewhere?

Reply:
That is a fair point, and you're right; it's handled by the request parser. The query parameter is a boolean. It'll either fail early or get through as `false`.